CondTools/CTPPS/src/WriteCTPPSPixelDAQMapping.cc

```diff
@@ -59,20 +59,19 @@ WriteCTPPSPixelDAQMapping::WriteCTPPSPixelDAQMapping(const edm::ParameterSet &ps
       tokenMapping_(esConsumes<CTPPSPixelDAQMapping, CTPPSPixelDAQMappingRcd>(edm::ESInputTag("", label_))) {}
 
 void WriteCTPPSPixelDAQMapping::analyze(const edm::Event &, edm::EventSetup const &es) {
-  // get DAQ mapping
-  edm::ESHandle<CTPPSPixelDAQMapping> hMapping = es.getHandle(tokenMapping_);
-
   // print mapping
   /*printf("* DAQ mapping\n");
   for (const auto &p : mapping->ROCMapping)
     cout << "    " << p.first << " -> " << p.second << endl;
   */
 
   // Write DAQ Mapping to sqlite file:
-  const CTPPSPixelDAQMapping *pCTPPSPixelDAQMapping = hMapping.product();  // DAQ Mapping
+
+  const auto mapping = &es.getData(tokenMapping_);
```

Comment:
@tvami @wpcarvalho I think that here I should write
const auto& mapping = es.getData(tokenMapping_)
as in PR#36062